GeosCore/fullchem_mod.F90

```diff
@@ -17,6 +17,11 @@ MODULE FullChem_Mod
 !
   USE Precision_Mod
 
+#ifdef MODEL_GCHP
+  USE MPI
+  USE, INTRINSIC :: ISO_C_BINDING
+#endif
```

Comment:
As @lizzlel stated we should change this to a separate #ifdef so that we can turn it off for testing.  Maybe:
```F90
#ifdef MODEL_GCHP
#ifdef MPI_LOAD_BALANCE
... etc ...

#endif
#endif
```
@lizziel, any thoughts on the name of the C-preprocessor switch?